tests/test_cli_compile.py

```diff
@@ -519,7 +523,7 @@ def test_generate_hashes_verbose(pip_conf, runner):
 
 
 @pytest.mark.skipif(PIP_VERSION < (9,), reason="needs pip 9 or greater")
-def test_filter_pip_markers(runner):
+def test_filter_pip_markers(pip_conf, runner, from_line):
```

Comment:
```suggestion
def test_filter_pip_markers(pip_conf, runner):
```
`from_line` could be deleted since it's not used in this test anymore